packages/nc-gui/components/smartsheet/grid/Table.vue

```diff
@@ -1732,7 +1732,7 @@ onKeyStroke('ArrowDown', onDown)
               v-if="contextMenuTarget && hasEditPermission"
               class="nc-base-menu-item"
               data-testid="context-menu-item-paste"
-              :disabled="isSystemColumn(fields[contextMenuTarget.col])"
+              :disabled="!!isSystemColumn(fields[contextMenuTarget.col])"
```

Comment:
Why adding `!!` to the function caller multiple times instead of adding `!!` to the function itself just once?

Reply:
To fix the following warning:
```js
[Vue warn]: Invalid prop: type check failed for prop "disabled". Expected Boolean, got Number with value 0.
```